apps/sms/views/shared/test/unit/navigation_test.js

```diff
@@ -657,6 +657,8 @@ suite('navigation >', function() {
 
       window.Navigation = NavigationFactory(fakeWindow);
       navigator.mozHasPendingMessage.withArgs('notification').returns(false);
+
+      Navigation.setReady();
```

Comment:
Please, tell me if you think we need any other dedicated test for this change, I can't really think of any reasonable test :/